### PR TITLE
More updates for 2.0

### PR DESCRIPTION
**Pull request by Sfshaza**
No redirects this time!

Staged: https://sz-webdev-1.firebaseapp.com/

### CODE REVIEW

**Comment by kwalrath** [comment]
Since this is similar to dart-lang/site-www#344 and might change based on what we do there, let's get that one settled before this one.

**Comment by kwalrath** [comment]
Taking a look now...

File: src/_codelabs/ng2/0-setup.md [hunk 1]
```diff
@@ -11,11 +11,17 @@ header:
...
+{% comment %}
+update-for-dart-2
+{% endcomment %}
+
 In this step, you install any software that you need,
 and learn where to find the sample code.
 
 ## <i class="fa fa-anchor"> </i> Get Dart and Dartium.
 
+{% include dartium-2.0.html %}
```
Comment (kwalrath) [inline src/_codelabs/ng2/0-setup.md]: Let's move this down to the bottom of this section (below "You will need Dartium...").

File: src/faq.md [hunk 2]
```diff
@@ -6,12 +6,8 @@ description: "FAQ and other tips for using Dart for web development."
 ---
 
 This FAQ applies to web programming. For more general Dart questions,
-see the [FAQ]({{site.dartlang}}/faq), or the [Tools FAQ]({{site.dartlang}}/tools/faq),
-both on dartlang.
-{% comment %}
-[TODO: Search for dart2js and Dartium.
-Update for dartdevc+Chrome, link to dartdevc FAQ.]
-{% endcomment %}
+see the [FAQ]({{site.dartlang}}/faq),
```
Comment (kwalrath) [inline src/faq.md]: remove the comma at the end of this line (before "or")

File: src/faq.md [hunk 3]
```diff
@@ -6,12 +6,8 @@ description: "FAQ and other tips for using Dart for web development."
...
+or the [Tools FAQ]({{site.dartlang}}/tools/faq), both on dartlang.
```
Comment (kwalrath) [inline src/faq.md]: dartlang -> www.dartlang.org?

(dartlang is not clear enough)

File: src/faq.md [hunk 4]
```diff
@@ -326,16 +318,9 @@ should have this kind of flexibility.
...
-
-#### Q. Why doesn't Dart support IE9 or earlier?
+Dart is designed to compile to JavaScript to run across the modern web.
 
-Supporting legacy browsers takes a lot of engineering resources and testing infrastructure.
-Dart is a bet for the future, and the project can't push forward if it needs to
-spend valuable resources on supporting browsers that are dying or dead.
-Also, dart2js can emit efficient code if it assumes a modern browser with
-ECMAScript5 or greater features.
+{% include dartium-2.0.html %}
```
Comment (kwalrath) [inline src/faq.md]: Remove this include? (It doesn't seem directly related to the question, and you have it higher up in this page.)

Comment (Sfshaza) [inline src/faq.md]: OK, this is weird. I can't find that text anywhere in src/faq.md, though I don't recollect deleting it.

**Pull request merged**